Bugsnag/BugsnagCrashSentry.m

```diff
@@ -85,6 +86,9 @@ - (void)reportUserException:(NSString *)reportName
                                        eventOverrides:eventOverrides
                                              metadata:metadata
                                                config:config];
+    
+    bsg_log_debug(@"Saved KSCrashReport for \"%@\" \"%@\"", handledState[@"severityReasonType"],
+                  [[eventOverrides[@"exceptions"] firstObject] valueForKey:@"errorClass"]);
```

Comment:
If `firstObject` returns `nil` or has no value will this be ok?

Reply:
If `firstObject` returns `nil` then this will not crash; it's fine to send a message to `nil` in Objective-C.